packages/@lwc/engine/src/framework/decorators/api.ts

```diff
@@ -37,11 +37,10 @@ export function createPublicPropertyDescriptor(key: string): PropertyDescriptor
             const vm = getAssociatedVM(this);
             if (isBeingConstructed(vm)) {
                 if (process.env.NODE_ENV !== 'production') {
-                    const name = vm.elm.constructor.name;
                     logError(
-                        `\`${name}\` constructor can’t read the value of property \`${toString(
+                        `Can’t read the value of property \`${toString(
```

Comment:
The error message had the constructor name and was convenient to understand. With this change, the user has to look at the stack trace to analyze the issue. Is there any issue with using `vm.def.ctor.name` to get the constructor name and print it in the message?

Reply:
This doesn't exist in SSR